tensorrt_llm/_torch/visual_gen/models/hunyuan_video1_5/transformer_hunyuan_video1_5.py

```
@@ -0,0 +1,1211 @@
+        timestep_r: Optional[torch.Tensor] = None,
+        encoder_hidden_states_2: Optional[torch.Tensor] = None,
+        encoder_attention_mask_2: Optional[torch.Tensor] = None,
+        image_embeds: Optional[torch.Tensor] = None,
+        attention_kwargs: Optional[Dict[str, Any]] = None,
+        return_dict: bool = True,
+    ) -> Union[Tuple[torch.Tensor], Transformer2DModelOutput]:
+        """
+        Args:
+            timestep: Normalized scheduler timestep tensor in [0, 1].
+        """
+        batch_size, num_channels, num_frames, height, width = hidden_states.shape
```

Ruff (0.16.1)

[warning] Unpacked variable `num_channels` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)